working-open.md

```diff
@@ -0,0 +1,46 @@
+# Working Open
+
+Hypha operates with democratic organizing principles informed by our [values](./values.md). As part of that we've developed a set of guidelines for how (and exactly what) and why we work in the open.
```

Comment:
> (and exactly what)

Not sure how to read this. Do you mean the tools/spaces/docs we keep open? If so, might "(and where)" capture the sentiment more clearly?

Reply:
```suggestion
Hypha operates with democratic organizing principles informed by our [values](./values.md). As part of that, we've developed a set of guidelines for how (and exactly what) and why we work in the open.
```